lib/index.js

```diff
@@ -183,3 +184,203 @@ exports.directory = FileSystem.directory;
  * Create a symbolic link factory.
  */
 exports.symlink = FileSystem.symlink;
+
+/**
+ * Perform action, bypassing mock FS
+ * @example
+ * // This file exists on the real FS, not on the mocked FS
+ * const filePath = '/path/file.json';
+ * const data = mock.bypass(() => fs.readFileSync(filePath, 'utf-8'));
+ */
+exports.bypass = function(fn) {
+  if (typeof fn !== 'function') {
+    throw new Error(`Must provide a function to perform for mock.bypass()`);
+  }
+
+  // Deactivate mocked bindings
+  const binding = realBinding._mockedBinding;
+  delete realBinding._mockedBinding;
+
+  let res;
+  try {
+    // Perform action
+    res = fn();
+  } finally {
+    // Reactivate mocked bindings
+    realBinding._mockedBinding = binding;
+  }
+
+  return res;
+};
+
+/* ****************************************************************************************************************** *
+ * Mapping
+ * ****************************************************************************************************************** */
+
+const mapping = {};
```

Comment:
Instead of tacking properties onto this object, it would be more consistent with the rest of the repo to create a new module that `exports` the functions you want to expose.  Is there a reason these functions need to be defined in this same scope?

Reply:
I certainly prefer it that way! :) 

The items in question are helper functions meant to be used with functions that belong to the `mock` object. I attached them to the object to prevent polluting the module scope. 

I had planned on creating a new file, but because the exported functions are meant to be attached to `mock`, judging by the rest of the repo, I felt it may be breaking convention to create a new file for them. So this was done in effort to be courteous. I'm glad to move them to their own file.